packages/codec/tests/molecule.test.ts

```diff
@@ -214,3 +215,225 @@ test("test fixed hex bytes", (t) => {
   t.truthy(hexStr === createFixedHexBytesCodec(3).unpack(hexBytes));
   t.throws(() => createFixedHexBytesCodec(4).pack(hexStr));
 });
+test("test simple array codec error", (t) => {
+  const codec = array(Uint8, 3);
+  try {
+    codec.pack([0x1, 0xfff, 3]);
+    t.fail();
+  } catch (e) {
+    t.truthy(
+      (e as CodecExecuteError).message.includes(
+        "Expect type Uint8BE at input[1] but got error: Value must be between 0 and 255, but got"
```

Comment:
Uint8 only contains 1 byte, so BE and LE is the same for it. Can you add some other test cases for Uints, like `Uint32LE`

Reply:
OK. I also add some cases for bigger units.